spot-client/src/common/i18n/en.json

```diff
@@ -31,6 +31,7 @@
         "pleaseWait": "Please wait",
         "reconnecting": "Reconnecting",
         "setup": "Currently in setup.",
+        "tvConflict": "Please make sure another window is not open and try again.",
```

Comment:
Maybe this should be more specific and call out to check if another instance of the same Spot TV is open? And that only one can be open at a time?

Reply:
Can you give me the exact text you have in mind? it will be easier this way

Reply:
Oh can we get the room name by this point? What if we do something like `{{ roomName}} is already running elsewhere. Please close the running instance and try again.`

Reply:
We should have the room name. I'll try to add that.